app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,17 @@
+class RelationshipsController < ApplicationController
+  def create
+    user = User.find(params[:user_id])
```

Comment:
No, because we're referencing it correctly. Strong params protects against passing all of `params[:user]` to `User.create`, because what if `params[:user]` contains `admin: true`? But when passing specific params, we don't need it.

From another perspective, if every time we used params we were explicit and did this:

```ruby
user = User.create(
  name: params[:user][:name],
  age: params[:user][:age],
  username: params[:user][:username],
)
```

...then we wouldn't need strong params at all.